Xamarin.Essentials/Calendar/Calendar.android.cs

```diff
@@ -0,0 +1,159 @@
+                CalendarContract.Events.InterfaceConsts.Dtend
+            };
+
+            // Android event ids are always integers
+            if (!int.TryParse(eventId, out var resultId))
+            {
+                throw new ArgumentException($"[Android]: No Event found for event Id {eventId}");
+            }
+
+            var calendarSpecificEvent = $"{CalendarContract.Events.InterfaceConsts.Id}={resultId}";
+            using (var cur = Platform.AppContext.ApplicationContext.ContentResolver.Query(eventsUri, eventsProjection.ToArray(), calendarSpecificEvent, null, null))
+            {
+                if (cur.Count > 0)
+                {
+                    cur.MoveToNext();
+                    var eventResult = new DeviceEvent
+                    {
+                        Id = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Id)),
+                        CalendarId = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.CalendarId)),
+                        Title = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Title)),
+                        Description = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Description)),
+                        Location = cur.GetString(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.EventLocation)),
+                        StartDate = DateTimeOffset.FromUnixTimeMilliseconds(cur.GetLong(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Dtstart))),
+                        EndDate = cur.GetInt(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.AllDay)) == 0 ? (DateTimeOffset?)DateTimeOffset.FromUnixTimeMilliseconds(cur.GetLong(eventsProjection.IndexOf(CalendarContract.Events.InterfaceConsts.Dtend))) : null,
+                        Attendees = GetAttendeesForEvent(eventId)
+                    };
+                    return eventResult;
+                }
+                else
+                {
+                    throw new ArgumentException($"[Android]: No Event found for event Id {eventId}");
+                }
+            }
+        }
+
+        static IEnumerable<DeviceEventAttendee> GetAttendeesForEvent(string eventId)
+        {
+            var attendeesUri = CalendarContract.Attendees.ContentUri;
+            var attendeesProjection = new List<string>
+            {
+                CalendarContract.Attendees.InterfaceConsts.EventId,
+                CalendarContract.Attendees.InterfaceConsts.AttendeeEmail,
+                CalendarContract.Attendees.InterfaceConsts.AttendeeName
+            };
+            var attendeeSpecificAttendees = $"{CalendarContract.Attendees.InterfaceConsts.EventId}={eventId}";
+            var cur = Platform.AppContext.ApplicationContext.ContentResolver.Query(attendeesUri, attendeesProjection.ToArray(), attendeeSpecificAttendees, null, null);
```

Comment:
Add using statement

Reply:
Addressed in: https://github.com/builttoroam/Essentials/pull/43